### DISCUSSION TITLE
Forward emails to any address to another address?

### DISCUSSION THREAD

**Discussion by nickshanks347** [docker-mailserver/docker-mailserver#3420] [Q&A]
I have several defined addresses, for example:

1. john@example.com

Is there a configuration settings that says to docker-mailserver that any addresses that isn't specifically setup should just forward any emails to a defined address?

So for example if an email was sent to xyzzy@example.com, or 138472@example.com that it should just forward to john@example.com.

But for any actual defined addresses, they shouldn't be forwarded to another address, they should just be sent as normal.

**Comment by casperklein** [comment]
[Catch-All](https://docker-mailserver.github.io/docker-mailserver/latest/faq/#how-can-i-configure-a-catch-all) is probably what you are looking for.

**Reply by polarathene** [reply]
That won't work with explicit accounts AFAIK: https://github.com/docker-mailserver/docker-mailserver/issues/3022#issuecomment-1610452561
